lib/active_data/model/validations.rb

```diff
@@ -18,6 +21,40 @@ def validate! context = nil
       def raise_validation_error
         raise ActiveData::ValidationError.new(self)
       end
+
+      module InstanceMethods
```

Comment:
Well, ActiveSupport::Concern is not used like this. You have simply put instance methods to the module itself